### PR TITLE
[develop] Update documentation about scaling up/down

### PR DESCRIPTION
**Pull request by rafzei**
Already merged to branch v1.2: https://github.com/epiphany-platform/epiphany/pull/2640

### CODE REVIEW

**Review by atsikham — DISMISSED** [review]

**Comment by seriva** [comment]
First we need to merge #2645.

File: docs/home/howto/CLUSTER.md [hunk 1]
```diff
@@ -440,17 +462,17 @@ To setup the cluster do the following steps from the provisioning machine:
       default_os_image: default
     ```
 
-    The [region](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html) lets you chose the most optimal place to deploy your cluster. The `subscription_name` is the Azure subscribtion under which you want to deploy the cluster.
+    The [region](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html) lets you chose the most optimal place to deploy your cluster. The `subscription_name` is the Azure subscription under which you want to deploy the cluster.
```
Comment (plirglo) [inline docs/home/howto/CLUSTER.md:465]: “Optimal,” already contains the concept of being the most suitable or desirable thing. So saying “most optimal” would be like saying “most most desirable" :)

File: docs/home/howto/CLUSTER.md [hunk 2]
```diff
@@ -440,17 +462,17 @@ To setup the cluster do the following steps from the provisioning machine:
...
+    Terraform will ask you to sign in to your Microsoft Azure subscription when it prepares to build/modify/destroy the infrastructure on `azure`. In case you need to share cluster management with other people you can set the `use_service_principal` tag to true. This will create a service principle and uses it to manage the resources.
 
     If you already have a service principle and don't want to create a new one you can do the following. Make sure the `use_service_principal` tag is set to true. Then before you run `epicli apply -f yourcluster.yml` create the following folder structure from the path you are running Epicli:
 
     ```shell
     /build/clustername/terraform
     ```
 
-    Where the clustername is the name you specified under `specification.name` in your cluster yaml. Then in the terraform folder add the file named `sp.yml` and fill it with the service priciple information like so:
+    Where the `clustername` is the name you specified under `specification.name` in your cluster yaml. Then in `terraform` folder add the file named `sp.yml` and fill it with the service principle information like so:
```
Comment (plirglo) [inline docs/home/howto/CLUSTER.md]:
```suggestion
    Where the `clustername` is the name you specified under `specification.name` in your cluster yaml. Then in `terraform` folder add the file named `sp.yml` and fill it up with the service principal information like so:
```

File: docs/home/howto/CLUSTER.md [hunk 3]
```diff
@@ -830,6 +869,13 @@ Note that clustering requires a change in the `configuration/rabbitmq` document:
   ...
   ```
 
+- RabbitMQ K8s:
+  Scaling is controlled via replicas in StatefulSet. Rabbitmq on K8s uses plugin rabbitmq_peer_discovery_k8s to works in cluster.
```
Comment (plirglo) [inline docs/home/howto/CLUSTER.md]:
```suggestion
  Scaling is controlled via replicas in StatefulSet. RabbitMQ on K8s uses plugin rabbitmq_peer_discovery_k8s to works in cluster.
```

**Review by plirglo — CHANGES_REQUESTED** [review]

File: docs/home/howto/CLUSTER.md (hunk 1 above)
Comment (rafzei) [reply docs/home/howto/CLUSTER.md:465]: True but its not mine! :cat:

**Review by atsikham — APPROVED** [review]

**Review by mkyc — APPROVED** [review]

**Review by plirglo — APPROVED** [review]

**Pull request merged**